tests/java_definitions.py

```diff
@@ -0,0 +1,28 @@
+import os
+from pathlib import Path
+
+import jpype
+import jpype.imports
+from jpype.types import *
+
+REPO_DIR = Path(__file__).parent.parent
+calcite_jars = Path.joinpath(REPO_DIR, "jars/*")
+
+the_java_home = "CONDA_PREFIX"
+if "JAVA_HOME" in os.environ:
+    the_java_home = "JAVA_HOME"
+
+java_home_path = os.environ[the_java_home]
+jvm_path = java_home_path
+
+if not os.path.isfile(jvm_path):
+    jvm_path = java_home_path + "/lib/libjli.dylib"
+
+jpype.startJVM("--enable-preview", convertStrings=True, jvmpath=jvm_path)
+jpype.addClassPath(calcite_jars)
+from com.google.protobuf.util import JsonFormat
+
+ArrayListClass = jpype.JClass("java.util.ArrayList")
+ListClass = jpype.JClass("java.util.List")
+SqlToSubstraitClass = jpype.JClass("io.substrait.isthmus.SqlToSubstrait")
+json_formatter = JsonFormat
```

Comment:
This seems odd.  I'm not sure we need to alias and reexport something from protobuf.  Can't the places that need this do their own `from com.google.protobuf.util import JsonFormat as json_formatter`? (although the alias is fishy too, is this a class or a method?)

Reply:
This is a class.  I can move it to context.py, where it's used.